src/Microsoft.Extensions.DependencyModel/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for reading `.deps` files.",
-  "version": "1.0.0",
+  "version": "1.0.0-preview2-1-*",
```

Comment:
This is actually decreasing the version number, which is not something we want to do.  An official version number is always HIGHER than the same version number with a pre-release.

This is a little tricky to manage since there are 3 places where this library is being maintained:

- preview2 branch
- preview2.1 branch
- core-setup (both master and release/1.1.0 branches)

I think to manage this, we should employ the following versioning strategy:

- preview2 branch gets versions `1.0.1` - `1.0.499`
- preview2.1 branch gets versions `1.0.500` - `1.0.999`
- core-setup gets versions `1.1.0`+

That gets us roughly 500 shippable builds from each branch.